.github/workflows/ubuntu.yml

```
@@ -126,3 +126,80 @@ jobs:
       run: |
         export OMP_NUM_THREADS=1
         mpirun -n 2 Examples/Physics_applications/laser_acceleration/PICMI_inputs_3d.py
+
+  build_qed_table_tools:
+    name: Clang QED table tools
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.draft == false
+    env:
+      # On CI for this test, Ninja is slower than the default:
+      #CMAKE_GENERATOR: Ninja
+      # setuptools/mp4py work-around, see
+      #   https://github.com/mpi4py/mpi4py/pull/159
+      #   https://github.com/mpi4py/mpi4py/issues/157#issuecomment-1001022274
+      SETUPTOOLS_USE_DISTUTILS: stdlib
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: |
+        .github/workflows/dependencies/pyfull.sh
+    - name: build WarpX & QED tools
+      run: |
+        export CC=$(which clang)
+        export CXX=$(which clang++)
+        cmake -S . -B build_qed \
+         -DWarpX_MPI=ON \
```

We could turn those off for this test :)
```suggestion
         -DWarpX_APP=OFF \
         -DWarpX_LIB=OFF \
         -DWarpX_MPI=ON \
```

---

sure! thanks!

---

Oops, you run WarpX first. So you need `-DWarpX_APP=ON`.
But please see the comment below, I think you can integrate this into another test above and save compile time with it.